File: FullLessons/subject6/python/CalculateAverage.py
#!/usr/bin/env python3

# Scanner input = new Scanner(System.in);  # Not needed in Python

print("How many numbers do you want to average?")
count = int(input())

sum = 0
for i in range(1, count + 1):  # Python uses range() for loops
    print("Enter number " + str(i) + ":  ")
    number = int(input())
    print()
    sum += number

average = sum / count  # No need for explicit casting in Python

print("The average is: " + str(average))

File: java/Section6Additional/CalculateAverage.java
import java.util.Scanner;

public class CalculateAverage {

  public static void main(String[] args) {

    Scanner input = new Scanner(System.in);

    System.out.println("How many numbers do you want to average?");
    int count = input.nextInt();

    int sum = 0;
    for (int i = 1; i <= count; i++) {
      System.out.print("Enter number " + i + ":  ");
      int number = input.nextInt();
      System.out.println();
      sum += number;
    }

    double average = (double) sum / count;

    System.out.println("The average is: " + average);
  }
}
